erpnext/projects/doctype/task/task.py

```diff
@@ -203,6 +205,20 @@ def update_status(self):
 				self.db_set('status', 'Overdue', update_modified=False)
 				self.update_project()
 
+	def notify(self):
+		if not frappe.db.get_value("Projects Settings", None, "send_notifications_for_task"):
+			return
+
+		notification_doc = {
+			'type': 'Notify',
+			'document_type': self.doctype,
+			'subject': _("Task {0} has been updated.").format("<a href='{0}'>{1}</a>".format(self.get_url(), frappe.bold(self.name))),
+			'document_name': self.name,
+			'from_user': frappe.session.user
+		}
+
+		enqueue_create_notification(self.get_assigned_users(), notification_doc)
```

Comment:
Why are the same changes being added to both the files? Also adding docstrings would be a good idea

Reply:
- We need to send notification if either one is updated, hence both the doctypes have the code.
- The function is easy to understand, so skipping docstrings